src/views/navbar/index.js

```diff
@@ -135,21 +151,47 @@ class Navbar extends React.Component<Props> {
             )}
           </Head>
 
-          <Logo to="/">
+          <Logo
+            to="/"
+            aria-hidden
+            tabIndex="-1"
+            isHidden={this.state.isSkipLinkFocused}
```

Comment:
Apparently, you don’t whitelist style props that go on `<Link>` components, so they are added to the DOM element and throw a React warning… It also happens for the `loggedOut` prop in the logged out header. I didn’t try to fix it.

Reply:
Yeah should just be a matter of filtering them—those console logs have been annoying me for a long time so a fix for this would be amazing!

Reply:
I think https://github.com/withspectrum/spectrum/pull/2752/commits/a2645419d9f9e1f6a0e96f35cf16e65766dbd6f1 is the way to go.
I might be missing some `data-…` attributes (`data-cy`, maybe?), but that’s the only foolproof solution I know of.

Reply:
Oh. I’m realizing now I could probably whitelist all props starting with `data-` as they’re all valid 🤔.